sys/bloom/bloom.c

```diff
@@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
     }
 
     /* Allocate Bloom array */
-    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
+    if (!(bloom->a = calloc(1, ROUND(size)))) {
```

Comment:
But size of an item is indeed sizeof(uint8)?
On Aug 24, 2015 10:50 AM, "René Kijewski" <notifications@github.com> wrote:

> In sys/bloom/bloom.c
> <https://github.com/RIOT-OS/RIOT/pull/3699#discussion_r37730511>:
>
> > @@ -35,7 +35,7 @@ bloom_t *bloom_new(size_t size, size_t num_hashes, ...)
> >      }
> >
> >      /* Allocate Bloom array */
> > -    if (!(bloom->a = calloc(ROUND(size), sizeof(char)))) {
> > +    if (!(bloom->a = calloc(1, ROUND(size)))) {
>
> But the parameters were swapped, actually. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/3699/files#r37730511>.
>